activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/validation/magicnumber/FileMagicNumber.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2017-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.activiti.cloud.services.modeling.validation.magicnumber;
+
+import java.math.BigInteger;
+import java.util.Arrays;
+
+public class FileMagicNumber {
+
+    private String name;
+    private String string;
+    private byte[] bytes;
+    private int offset;
+
+    public String getString() {
+        return string;
+    }
+
+    public void setString(String string) {
+        this.string = string;
+        if (string != null) {
```

Comment:
could we enclose the logic of this setter into a private method giving it a descriptive name?
just to avoid the guess work when/if changing this logic in the future

Reply:
good catch! done